pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -138,7 +137,7 @@ const CryptoPolicyDialog = ({
     if (!(currentCryptoPolicy in cryptopolicies)) {
         policies.push({
             name: currentCryptoPolicy,
-            title: displayProfileText(currentCryptoPolicy),
+            title: currentCryptoPolicy,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18907-20230623-160111-2a44ca96-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)